### PR TITLE
Fix live stream style

### PR DESCRIPTION
**Pull request by mistermendez**

### CODE REVIEW

File: js/components/utils.js [hunk 1]
```diff
@@ -84,23 +84,25 @@ var Utils = {
   * @return {String} The time as a string in the HH:MM format
```
Comment (rchalooyala) [inline js/components/utils.js]: Should it be documented that nothing is returned if the timeInSeconds is not finite?

**Comment by mistermendez** [comment]
@rchalooyala removed isFinite check from Util.formatSeconds().

isFinite check is in controlBar time duration.

**Comment by rchalooyala** [comment]
yep, perfect

**Pull request merged**